### ISSUE TITLE
Enhanced MPD parsing

### ISSUE THREAD

**Issue by NickEastNL** [stargatefreak92/LEGO-3ds-Max-importer#1]
The assembler only assembles (sub)models. It does not duplicate and place submodels in their correct positions, due to the complexity of hierarchies in MPD files. As a workaround, the assembler creates point helpers and parents each part of the submodel to the helper. The submodels then have to be duplicated and placed manually.

To simplify this process, I intend to have the assembler create placeholder helpers where these submodels have to be placed. It is then only a matter of duplicating the submodels and using the "Align" tool in Max to align the submodel parent helpers to the placeholders.

**Comment by NickEastNL** [comment]
Basic placeholder handling implemented. Now I only have to clean up the code a bit so the final assembled model is also clean (e.g. don't have the assembler create a point helper for the main "root" helper)

**Comment by NickEastNL** [comment]
I cleaned up the code to remove redundant code. I've decided to let the assembler create a point helper for the "root" of the model. It would require a major refactor of the code (both C# and MaxScript) to prevent this, while maintaining the other functionality. Keeping the point helper also ensures that a proper hierarchy is maintained in Max's Outliner, so the benefits weigh more. A standard .ldr file does _not_ create point helpers; only multi-part document files do.
